src/core/tests/Test_ConfigurePatchingProcessor.py

```diff
@@ -317,6 +321,7 @@ def test_configure_patching_with_patch_mode_and_assessment_mode_by_platform(self
     def test_configure_patching_raise_exception_auto_os_patch_state(self):
         # arrange capture std IO
         captured_output = StringIO()
+        original_stdout = sys.stdout
```

Comment:
Checking...
Is this sys.stdout or sys.__stdout__? Since that was removed from line 342 below

Reply:
this is sys.stdout can be redirect and store terminal output